testutil/daemon/daemon.go

```diff
@@ -395,7 +400,10 @@ func (d *Daemon) Kill() error {
 		return err
 	}
 
-	return os.Remove(fmt.Sprintf("%s/docker.pid", d.Folder))
+	if d.pidFile != "" {
```

Comment:
Would be good to note in the commit message that this fixes bogus error from removing `docker.pid` that was never created.

Reply:
@SamWhited I would 
- not care about error returned from `os.Remove()` at all
- not return any error here as this will fail the test

I.e. current approach looks fine for me.